modules/server_processes_manager/README.md

```diff
@@ -0,0 +1,65 @@
+# Server Processes Manager
+
+## Purpose
+
+The server processes manager module is used to get information about processes that were 
+launched asynchronously by other LORIS modules
+
+## Intended Users
+
+The only user that should have access to this module is the person in charge of
+the administration of the LORIS website
+
+## Scope
+
+This module displays the information relevant to the asynchronous processes
+(active and inactive) launched by other modules. The properties displayed for
+each process are as follows:
+
+- The internal LORIS process ID for the process
+- Pid: the Unix process ID of the command that was launched asynchronously.
+- Type: type of process (only mri_upload is currently implemented).
+- Stdout file: full Unix path of the Output log file on the LORIS server 
+(STDOUT output stream for the process).
+- Stderr file: full Unix path of the Error log file on the LORIS server 
+(STDERR output stream for the process).
+- Exit code file: full Unix path of the file on the LORIS server that will 
```

Comment:
I'm not sure I understand why this is kept around when the next column is already there

Reply:
Could be useful to have the name of the file in which the exit code goes. I do agree though that in most cases you won't need this...